src/appsignal/cli/demo.py

```diff
@@ -78,20 +78,18 @@ def transmit() -> None:
                 "otel.instrumentation_library.name",
                 "opentelemetry.instrumentation.wsgi",
             )
-            span.set_attribute(
-                "demo_sample",
-                True,
-            )
+            set_tag("demo_sample", True, span)
 
         # Error sample
         with tracer.start_as_current_span("GET /demo") as span:
             span.set_attribute("http.method", "GET")
             set_params({"GET": {"id": 1}, "POST": {}}, span)
-            span.set_attribute(
-                "demo_sample",
-                True,
-            )
+            set_tag("demo_sample", True, span)
             try:
-                raise ValueError("Something went wrong")
-            except ValueError as e:
+                raise DemoError("Something went wrong")
+            except DemoError as e:
                 span.record_exception(e)
```

Comment:
nit: in the line of the other changes to use `set_tag` and `set_params`, this could be `set_error(e)`

Reply:
I've changed it in #166